### PR TITLE
[WIP] Feature/0.1.0

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
Leaving this open as a PR while work is in progress just so we get ongoing CI results for it.

@rynowak This is the former 0.5.0 branch, now rebased onto `dev`. For clarity I also deleted `feature/0.5.0`.

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
@rynowak Rebased on `dev` again

File: global.json [hunk 1]
```diff
@@ -1,5 +1 @@
-{
-  "sdk": {
-    "version": "2.1.3"
-  }
-}
+{ "sdk": { "version": "2.1.300-preview2-008322" } }
```
Comment (davidfowl) [inline global.json]: I don't think we check these in

Comment (rynowak) [reply global.json]: I do, I'm a renegade

**Comment by rynowak** [comment]
BTW @SteveSandersonMS - I've been holding off on merging this since the next preview of VS is required to run the VS tools.

**Comment by rynowak** [comment]
FYI 15.7 preview 1 is live now on the preview channel. That means that anyone out there in TV land could install 15.7 and our tools easily.

**Comment by SteveSandersonMS** [comment]
> FYI 15.7 preview 1 is live now on the preview channel

Hooray! I've merged this into `dev` now (after rebasing it again). So we're done with the `feature/0.1.0` branch now, and ongoing work can continue in `dev`.

**Pull request closed without merging**